packages/edit-site/src/components/template-details/index.js

```diff
@@ -32,7 +32,6 @@ export default function TemplateDetails( { template, onClose } ) {
 
 	// TODO: We should update this to filter by template part's areas as well.
 	const browseAllLinkProps = useLink( {
-		canvas: 'view',
```

Comment:
I think this causes regression of #48301.

Reply:
it should be "undefined" now because "view" is the default value. But I guess it suffers from the issue I talk about here https://github.com/WordPress/gutenberg/pull/48731#issuecomment-1453654827